### PR TITLE
Pull request for Get Started with Oracle Data Safe Fundamentals 

### PR DESCRIPTION
**Pull request by joglover**
WMS ID: 700

Added a SQL Firewall lab
Upgraded environment to 23ai to support SQL Firewall
Changes made for Tiger Team training tomorrow!

### CODE REVIEW

File: data-safe/sqlfw/sqlfw-deploy-sqlfw-policy.md
Comment (ramonamagadan18) [inline data-safe/sqlfw/sqlfw-deploy-sqlfw-policy.md:1]: Hi @joglover! The Acknowledgements section for this Lab is missing. Please make sure to include it. Thank you!

**Review by ramonamagadan18 — CHANGES_REQUESTED** [review]

File: data-safe/sqlfw/sqlfw-deploy-sqlfw-policy.md
Comment (joglover) [reply data-safe/sqlfw/sqlfw-deploy-sqlfw-policy.md:1]: Hi @ramonamagadan18 , the lab to which you are referring is not in use in any workshop. It is currently a work in progress.

Comment (joglover) [reply data-safe/sqlfw/sqlfw-deploy-sqlfw-policy.md:1]: I've added acknowledgement just so we can move forward :)

Comment (ramonamagadan18) [reply data-safe/sqlfw/sqlfw-deploy-sqlfw-policy.md:1]: Thank you!

**Review by ramonamagadan18 — APPROVED** [review]

**Pull request merged**